bin/bundle

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```

Comment:
These are rails generated files. When you run `rails app:update` as suggested in the upgrade guide, the update task automatically updates this to the latest code for these tools. I would suggest we keep it if possible since this will occur on every `rails app:update` run which might include further changes.

Reply:
Ah, I didn't notice the `bin/` path. Good call.